src/main/java/com/dev/cinema/config/AppConfig.java

```diff
@@ -0,0 +1,47 @@
+package com.dev.cinema.config;
+
+import java.util.Properties;
+import javax.sql.DataSource;
+import org.apache.commons.dbcp2.BasicDataSource;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.context.annotation.Bean;
+import org.springframework.context.annotation.ComponentScan;
+import org.springframework.context.annotation.Configuration;
+import org.springframework.context.annotation.PropertySource;
+import org.springframework.core.env.Environment;
+import org.springframework.orm.hibernate5.LocalSessionFactoryBean;
+
+@Configuration
+@PropertySource("classpath:db.properties")
+@ComponentScan(basePackages = {"com.dev.cinema"})
```

Comment:
it's better to set more specific packages here (dao, service, security). In other case problems mat occur (had yesterday on QnA issues because of that)